### ISSUE TITLE
Uncaught TypeError: postTitleVsIdMap is undefined

### ISSUE THREAD

**Issue by htmgarcia** [publishpress/PublishPress-Blocks#961]
1. Switch to Code editor
2. Exit code editor
3. Try to exclude a post through "Exclude these posts" field

**Comment by htmgarcia** [comment]
Error fixed, however the "Exclude these posts" field still doesn't allow to change its values after step 2. Opened a new issue #962
